src/views/virtualmachines/actions/components/DeleteVMModal/components/DeleteOwnedResourcesMessage.tsx

```diff
@@ -36,7 +41,7 @@ const DeleteOwnedResourcesMessage: FC<DeleteOwnedResourcesMessageProps> = ({
     );
   }
 
-  const pvcsWithNoDataVolumes = pvcs?.filter((pvc) => !findPVCOwner(pvc, dataVolumes));
+  const pvcsWithNoDataVolumes = pvcs?.filter((pvc) => !findPVCOwner(pvc, dataVolumes)) || [];
```

Comment:
why filtering disk with datavolume? we have a GC for datavolumes once PVC is bound and ready, what happen if disks in error stage and we want to delete the VM and disks? the PVC still has DV and should be deleted as well

Reply:
This is just to show the right amount of disks. It would be confusing to show two disks with the same name one for the DV and another for the PVC. 

The real logic behind the delete disks logic is to delete the VM owner reference for the resources that the user doesn't want to delete. THe other resources will be deleted because of owner reference deletion policy